lyricsheets/fonts/win32font.py

```diff
@@ -5,15 +5,23 @@
 import win32ui
 import win32con
 
+_mappings = {}
+
 class Win32Font:
+    def get_font(style: Style):
+        if style.name not in _mappings:
+            _mappings[style.name] = Win32Font(style)
+        
+        return _mappings[style.name]
+
```

Comment:
Just make the `get_font` method a part of the constructor and make `_mappings` a class-level variable?

Reply:
It's meant to be more of a factory method in this instance, so I don't keep on opening new Windows device contexts every time a font is needed.

Reply:
Fair enough, but label it as a `@staticmethod` then?

(Nitpick: The key of the map should really be a tuple of all the fields involved in font creation (fontName, fontSize, isBold, isItalic, isUnderline, isStrikeout). One way I can see using the name as key as potentially being a problem is for example, if two different files have styles defined with the same name (e.g. Default) but different parameters (e.g. font size))

Reply:
Updated both.